README.md

```diff
@@ -1,7 +1,6 @@
 # @loopback/example-shopping
 
 [![Travis Build Status](https://travis-ci.com/strongloop/loopback4-example-shopping.svg?branch=master)](https://travis-ci.com/strongloop/loopback4-example-shopping)
-[![Greenkeeper badge](https://badges.greenkeeper.io/strongloop/loopback4-example-shopping.svg)](https://greenkeeper.io/)
```

Comment:
RenovateBot does not support badges yet, see the discussion in https://github.com/renovatebot/renovate/issues/534